startleft/validators/schema.py

```diff
@@ -31,7 +32,7 @@ def json(self):
 
     def __load_schema(self, schema_file_name):
         schema_path = pkg_resources.resource_filename('startleft', os.path.join('data', schema_file_name))
-        logger.debug(f"Loading schema from {schema_path}")
```

Comment:
Take care. The load_schema method is called for both otm and mapping definition schemas